src/main/java/filemanager/model/Command.java

```diff
@@ -26,4 +26,7 @@
     @JsonProperty
     private String client;
 
+    @JsonProperty
+    private String passKey;
+
```

Comment:
I've misstyped the term. Please make passkey be PathParameter of the query. It shouldn't be part of the command.